examples/gba-example-rainbow/README.md

```diff
@@ -0,0 +1,21 @@
+# gba-example-rainbow
+
+An extremely basic test application using the `gba-core` library.
+
+![](./screenshot.png)
+
+## Dependencies
+
+- The `rust-src` Rust component (`rustup component add rust-src`)
+- `cargo-xbuild` and `cargo-binutils` (available via `cargo install`)
+- `gbafix` somewhere in your `$PATH` - Either the DevkitPro version, or the
```

Comment:
guess we don't need this now you've added a rust version?

Reply:
the rust version doesn't have options to change the ROM title or version or anything, yet. that's something i'll be implementing soon, and when it's in there, this can be removed